src/brogue/IO.c

```diff
@@ -4239,105 +4240,6 @@ void printDiscoveriesScreen() {
     overlayDisplayBuffer(rbuf, NULL);
 }
 
-// Creates buttons for the discoveries screen in the buttons pointer; returns the number of buttons created.
```

Comment:
I sometimes remove big blocks of commented code if they refer to variables that I bulk changed